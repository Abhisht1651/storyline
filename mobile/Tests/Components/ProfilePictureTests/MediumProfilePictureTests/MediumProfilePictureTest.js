import { shallow } from 'enzyme'
import React from 'react'
import MediumProfilePicture from '../../../../App/Components/ProfilePicture/MediumProfilePictures/MediumProfilePicture'

describe('MediumProfilePicture', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<MediumProfilePicture />)
      expect(component).toMatchSnapshot()
    })
  })
})
