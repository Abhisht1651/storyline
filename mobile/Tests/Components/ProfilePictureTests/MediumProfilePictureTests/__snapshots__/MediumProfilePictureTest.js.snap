// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediumProfilePicture Rendering should match to snapshot 1`] = `
<View>
  <ForwardRef
    activeOpacity={1}
    onPress={[Function]}
    style={
      Object {
        "alignItems": "center",
        "borderRadius": 100,
        "height": 88,
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "default": "",
        }
      }
      style={
        Object {
          "aspectRatio": 1,
          "borderRadius": 100,
          "height": "100%",
        }
      }
    />
  </ForwardRef>
</View>
`;
